src/Material/Textfield.elm

```diff
@@ -236,6 +265,27 @@ view' lift model options =
       Options.collect defaultConfig options
     val = 
       config.value |> Maybe.withDefault model.value
+
+    isTextarea = config.type' == type' "textarea"
```

Comment:
Does this actually work? Even if it did, it might stop doing so in the future. I'd suggest instead introducing a type

    type Kind 
      = Input
      | TextArea
      | Password

Then change `Config.type` to have type `Kind` (maybe rename it `kind` in the process), then switch on `config.kind` in the appropriate places in `view`.